src/Hl7.Fhir.Shims.STU3AndUp/ElementModel/TypedElementExtensions.cs

```diff
@@ -9,18 +9,47 @@
 #nullable enable
 
 using Hl7.Fhir.ElementModel.Adapters;
+using Hl7.Fhir.Introspection;
 using Hl7.Fhir.Model;
+using Hl7.Fhir.Utility;
+using System;
+using System.Diagnostics.CodeAnalysis;
 using System.Runtime.CompilerServices;
 
 namespace Hl7.Fhir.ElementModel
 {
     public static class TypedElementExtensions
     {
-        [TemporarilyChanged] // This should be restored to use the original ITE stack.
+        /// <summary>
```

Comment:
Feels like a complete duplication of the same file in Base, so maybe move it to Base shims.